packages/block-library/src/cover/edit.js

```diff
@@ -97,9 +113,20 @@ class CoverEdit extends Component {
 			dimRatio,
 			focalPoint,
 			hasParallax,
-			id,
 			url,
 		} = attributes;
+
+		const onSelectUrl = ( newURL ) => {
+			if ( newURL !== url ) {
+				this.props.setAttributes( {
```

Comment:
We are not setting the media type so, If I have a video as the cover, and I switch to use an image by URL things fail because the media type is still video.

Reply:
oups!